src/main/java/org/broadinstitute/hellbender/tools/exome/SNPSegmenter.java

```diff
@@ -16,23 +17,27 @@
     /**
      * Write segment file based on allelic counts at SNP sites.  Converts allelic counts to target coverages,
      * which are written to a temporary file and then passed to {@link RCBSSegmenter}.
-     * @param snpCounts             list of allelic counts at SNP sites
+     * @param snps                  TargetCollection of allelic counts at SNP sites
      * @param sampleName            sample name
      * @param outputFile            segment file to write to and return
-     * @throws IOException          if temporary target file cannot be created or written to
      */
-    public static void writeSegmentFile(final List<AllelicCount> snpCounts,
-                                        final String sampleName, final File outputFile)
-            throws IOException {
-        final File targetsFromSNPCountsFile = File.createTempFile("targets-from-snps", ".tsv");
+    public static void writeSegmentFile(final TargetCollection<AllelicCount> snps,
+                                        final String sampleName, final File outputFile) {
+        try {
+            final File targetsFromSNPCountsFile = File.createTempFile("targets-from-snps", ".tsv");
 
-        List<TargetCoverage> targetsFromSNPCounts = snpCounts.stream()
-                .map(count -> count.toMinorAlleleFractionTargetCoverage("snp-target" + count.getContig() + ":" +
-                        count.getStart() + "_" + count.getEnd())).collect(Collectors.toList());
+            List<TargetCoverage> targetsFromSNPCounts = snps.targets().stream()
+                    .map(count -> count.toMinorAlleleFractionTargetCoverage("snp-target" + count.getContig() + ":" +
+                            count.getStart() + "_" + count.getEnd())).collect(Collectors.toList());
```

Comment:
this is just an arbitrary target name so it doesn't matter, but i'll go ahead and change it